components/PowerInfoCard.tsx

```diff
@@ -0,0 +1,34 @@
+import { FC } from 'react';
+import Image from 'next/image';
+
+type PowerInfo = {
+  src: string;
+  applicance: string;
+  kWh: string;
+};
+
+const PowerInfoCard: FC<PowerInfo> = ({ src, applicance, kWh }) => {
+  return (
+    <div className="w-full">
```

Comment:
this `div` seems unnecessary—I think `w-full` can be moved into the styles of the `div` that is being wrapped.